static/queues.html

```diff
@@ -184,6 +230,67 @@ <h3>Add queue</h3>
       document.querySelector('#dataTags').onclick = e => {
         window.avalanchemq.helpers.argumentHelperJSON("arguments", e)
       }
+
+      document.querySelector("#deleteQueues").addEventListener('submit', function (evt) {
+        evt.preventDefault()
+
+        if (window.confirm('Are you sure? The selected queues are going to be deleted. Messages cannot be recovered after deletion.')) {
+          var selectedList = document.querySelectorAll('[data-select]')
+          var vhost; 
+          var name; 
+
+          for (let i = 0; i < selectedList.length; i++) {
+            vhost = selectedList[i].dataset.vhost
+            name = selectedList[i].dataset.name
+            const url = '/api/queues/' + vhost + '/' + name
+            avalanchemq.http.request('DELETE', url)
+            .then(() => { window.location = '/queues' })
```

Comment:
won't this cause a page reload, for each delete?

Reply:
We talked about this and it will be changed so that the pages isn't reloaded at all.